### PR TITLE
website:fix validation failures

### PR DESCRIPTION
**Pull request by jgil-r**
In install option links, and feature comparison
table I fixed aria-labelled-by attribute to correct spelling
aria-labelledby. Fixed time element to have correct datetime attribute,
which translates dates into machine-readable format. Lastly, changed
obsolete tt element, in favor of code element. #335

### CODE REVIEW

**Comment by jgil-r** [comment]
I have read and hereby agree to quick-lint-js' Contributor License Agreement (CLA-v1.md).

File: website/public/index.ejs.html [hunk 1]
```diff
@@ -150,7 +153,7 @@ <h2>Install quick-lint-js</h2>
               class="install-logo"
               src="gnome-terminal.svg"
               alt=""
-              aria-labeled-by="install-cli"
+              aria-labelledby="install-cli"
```
Comment (strager) [inline website/public/index.ejs.html:156]: 🤦‍♀️

**Review by strager — APPROVED** [review]
Looks good! Shall I merge it now?

**Comment by strager** [comment]
> Removed invalid aria-labelledby attribute from img tags in install
options links, and added alt attribute instead.

This description doesn't match the code.

**Comment by jgil-r** [comment]
> > Removed invalid aria-labelledby attribute from img tags in install
> > options links, and added alt attribute instead.
> 
> This description doesn't match the code.

Yeah I just fixed it actually. I forgot to rebuild the site before the latest commit though, is that why there is one failing check?

**Comment by strager** [comment]
> I forgot to rebuild the site before the latest commit though, is that why there is one failing check?

No. The CentOS check failure is unrelated to your PR. It's happening on master. (I just noticed it.)

**Comment by jgil-r** [comment]
Okay, it should be good to go then. Would you like me to edit the main comment with a description of the new changes from most recent commit?

**Comment by strager** [comment]
> Okay, it should be good to go then. Would you like me to edit the main comment with a description of the new changes from most recent commit?

Yes. This message will serve as the published commit's message.

**Comment by jgil-r** [comment]
Done, should be good now.

**Pull request merged**